src/index.tsx

```diff
@@ -12,7 +12,8 @@ import App from './App';
 import * as serviceWorker from './serviceWorker';
 
 ReactDOM.render(
-  <React.StrictMode>
+  // <React.StrictMode>
```

Comment:
I noticed before in a different project that React.StrictMode allows for render to happen twice in development for some reason. However, I wasn't totally sure if that behavior was consistent here too, so I can undo these deletions if need be.

Reply:
We need to have React.StrictMode. There is a lot of good reason why we need it. Check https://reactjs.org/docs/strict-mode.html. We should not be commenting it.